jujugui/static/gui/src/app/components/profile/_profile.scss

```diff
@@ -1,62 +1,70 @@
 .profile {
-  &__content {
-    display: grid;
-    grid-template-columns: 238px 1fr;
+
+  .icon-list {
+    margin-bottom: 0;
   }
 
-  a,
-  .link {
-    color: $link-color;
-    font-weight: 200;
+  &__main {
+    padding-top: 0rem !important;
```

Comment:
This is required because of the `.v1` prefix on the import bumps the specificity of v1 styles by an order of magnitude meaning in certain situations like this where the styles need to be over-ridden, !important is required. The alternative is to refactor CSS selectors to prepend parent elements but this has the significant downside of tightly coupling the stylesheet to the DOM.

These styles sit at the end of cascade as such should not need to be overwritten hence why `!important` is the best choice in this case.

Reply:
Sounds good